packages/field-plugin/src/createFieldPlugin/createPluginActions/partialPluginStateFromStateChangeMessage.ts

```diff
@@ -17,7 +18,7 @@ export const partialPluginStateFromStateChangeMessage = (
   token: message.token ?? undefined,
   options: recordFromFieldPluginOptions(message.schema.options),
   uid: message.uid ?? undefined,
-  content: message.model ?? undefined,
+  content: parseContent(message.model),
```

Comment:
This is an improvement, because now `content` is always adhering to the user's defined schema.